packages/block-library/src/navigation-menu-item/block.json

```diff
@@ -12,9 +12,15 @@
 		"title": {
 			"type": "string"
 		},
+		"type": {
+			"type": "string"
+		},
 		"description": {
 			"type": "string"
 		},
+		"link_id": {
```

Comment:
Does this need to have an underscore?

Reply:
It shouldn't as it doesn't follow the pattern used with other attributes.